tox.ini

```diff
@@ -103,7 +103,9 @@ passenv =
 
 [testenv:docs]
 whitelist_externals = make
-deps = sphinx
+deps =
+  pydeps
```

Comment:
We need to ask @acozine/@gundalow how the things are installed in Jenkins as it doesn't run tox atm AFAIK.

Reply:
Jenkins job was disabled, likely because it was failing and nobody noticed or fixed it. It can be reinstated (though probably not until after the 2.10 beta is cut).

Reply:
@acozine can we move that job to GHA?